x-pack/plugins/alerting/server/saved_objects/migrations.test.ts

```diff
@@ -699,6 +699,282 @@ describe('7.11.2', () => {
   });
 });
 
+describe('7.13.0', () => {
```

Comment:
We've had issues in the past using `undefined` as param values for actions, when we "updated" the SO.  Basically, if the SO param value currently has a value, and then you "update" the SO with new params where the param value is changed to `undefined`, it's a no-op in terms of changing that param value, since "update" is actually a partial update.

I believe we've fixed these issues - we no longer use "update" - and I think we only saw this for actions anyway, but I think it would be worth an extra test.  By adding a FT which creates one of these alerts with a non-undefined param, then updates the alert with the param now being undefined, then doing a GET on the alert to make sure the param got "deleted" (has no value).  Needs to be a FT so it can walk all the way through ES and back, each time.

Unless there are already some tests like that for the alert params in the FTs for this alert.

Reply:
We have API integration tests that cover this case, e.g. https://github.com/elastic/kibana/blob/master/x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules.ts#L156

It looks to me like the issue with `undefined` values not replacing existing values was fixed as part of https://github.com/elastic/kibana/pull/73688